### DISCUSSION TITLE
The current naming method is actually super useful + 1 setting request.

### DISCUSSION THREAD

**Discussion by delano8** [aclap-dev/video-downloadhelper#189] [Help and Q&A]
### Your question

I've seen some people complain about the current "broken" naming method. It's actually fantastic and I wish this would stay available (as a setting). Why?

With the previous method, for instance I'm downloading 100 videos from one page. This page names them all the same. This means the videos are called video(1) - video(100) in my download folder if I download them all in 1 sitting.
But, let's say I split up my download sessions, and forget where I was. It will just keep on naming them video(51) etc but maybe 51 is actually the same as 46. I have no clue if I double download some of the files.
The current "broken" naming method gives them all unique names. So I can start downloading around where I think I was, then copy/paste over the previous "load" and see which are double (because the unique names get detected and I can skip those). That's great!
Please keep this available, it's super useful for pages where every file is named the same!

1 more request, can we get a setting to flip the hits list? Previously the newest/latest one was at the top, now, it's at the bottom. This is bad if you want to download 10+ things and after a while you have to scroll all the time, which wasn't the case before.
Would be great if this was a setting.

### Addon & CoApp version and details

version: 8.2.0.23
channel: stable
build date: 2024-02-14
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: unset
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:123.0) Gecko/20100101 Firefox/123.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.10",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\Delano",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "6.0"
}

**Comment by paulrouget** [comment]
Do you the CoApp or the browser to download?

> can we get a setting to flip the hits list

Noted! Might do that for 8.3.

**Reply by delano8** [reply]
Using the Firefox addon. Before the last update, every file that was names the same on a site, got name(1) etc, so this method of naming is way better (for me).

And the list has been newest on top for a long time I think? And now it's the opposite. Which causes unnecessary scrolling.
